src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -183,6 +183,14 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "DELETE" }
     ),
+    deleteArtworkImagesLoader: gravityLoader<
+      any,
```

Comment:
out of curiosity what does this `any` expect?

Reply:
Our path generation seems to understand that we have more than o1ne parameter if we create it this way. In cases where we only have 1 parameter, that line is something like:

```TypeScript
(id) => `partner/${id}/artworks`,

Reply:
There are a lot of layers of abstraction here, but I was feeling like adventuring to learn more about metaphysics. So I went on an adventure to figure out what that first type arg refers to.

This is the signature used by that line: https://github.com/artsy/metaphysics/blob/2fceae7a655fb1f337de5527aa3df27138446107/src/lib/loaders/index.ts#L47-L51

So that first type arg is the type of the `Body`. I still wasn't sure what this meant, so I found an example where the first type arg is being specified:

https://github.com/artsy/metaphysics/blob/2fceae7a655fb1f337de5527aa3df27138446107/src/lib/loaders/loaders_without_authentication/gravity.ts#L193-L199

It's using a different signature than our example, but the first type argument aligns with our signature, so it's still a good example to look at.

The shape of `GravityCalculatedCostResponse` looks like this: https://github.com/artsy/metaphysics/blob/2fceae7a655fb1f337de5527aa3df27138446107/src/lib/loaders/loaders_without_authentication/gravity.ts#L20-L28

But how is it used? I looked for usage of `saleArtworkCalculatedCostLoader ` to see where that type shows up: 

https://github.com/artsy/metaphysics/blob/2fceae7a655fb1f337de5527aa3df27138446107/src/schema/v2/sale_artwork.ts#L321-L357

This is where the type shows up! Line 334 of the resolver calls the `saleArtworkCalculatedCostLoader`, and its response gets typed as a `GravityCalculatedCostResponse`, which makes processing it within the resolver pretty nice.

## What's this mean for our changes?

If we were to use a strong type instead of `any` on this line, we would get nice type safety on the `response` from the `deleteArtworkImageLoader` here: 
 https://github.com/artsy/metaphysics/blob/2fceae7a655fb1f337de5527aa3df27138446107/src/schema/v2/me/myCollectionDeleteArtworkImageMutation.ts#L37-L47

## What's this mean for this PR? 

I'm not sure if it's a lot of effort to make a type for that response, but if it is not a ton, I think it'd be valuable to have ours as another example of how to use a strong type argument in that signature. There are significantly more examples in the codebase of using `any` than there are of using a strong type, and if we can nudge it in the direction of more type-safety than I"m all for it. 

Having said that, I **would not** hold up a merge for that.

Reply:
If possible, might be worth trying to replace `any` for `null`, but either way nbd 👍

Reply:
(Nice research, I wasn't even aware we had that typed pattern in MP)

Reply:
Fascinating! Great sleuthing. I'm 👍 on leaving the `any` on the response body typing.